main.py

```diff
@@ -790,6 +800,9 @@ def add_links(self, event):
             return
 
         if created:
+            self.update_instance_metadata(link)
+            link.update_metadata('last_status_is_active', True)
```

Comment:
When I was fixing this I've also mapped this one here https://github.com/kytos-ng/kytos/issues/165, it's nothing new, the team is already aware of, but I've mapped there just so we can prioritize later and minimize the surface of race conditions